packages/wrangler/src/config/index.ts

```diff
@@ -138,6 +141,20 @@ export function printBindings(bindings: CfWorkerInit["bindings"]) {
 		});
 	}
 
+	if (d1_databases !== undefined && d1_databases.length > 0) {
+		output.push({
+			type: "D1 Databases",
+			entries: d1_databases.map(({ binding, database_name, database_id }) => {
+				return {
+					key: binding.slice(D1_BETA_PREFIX.length),
```

Comment:
Perhaps move this to a helper next to the `identifyD1BindingsAsBeta()` helper? Keep it all in one place?